eZ/Bundle/EzPublishCoreBundle/EventListener/RequestEventListener.php

```diff
@@ -76,9 +76,12 @@ public function onKernelRequestIndex( GetResponseEvent $event )
     {
         $request = $event->getRequest();
         $semanticPathinfo = $request->attributes->get( 'semanticPathinfo' ) ?: '/';
+        $siteaccess = $request->attributes->get( 'siteaccess' );
+        $isLegacyMode = $this->configResolver->getParameter( 'legacy_mode', 'ezsettings', $siteaccess->name );
```

Comment:
Good point!  I will double-check it but you should be right, no need to specify it.